### ISSUE TITLE
Some observations on the icesheet process

### ISSUE THREAD

**Issue by imagico** [fossgis/osmdata#1]
https://github.com/joto/osmdata/blob/b5f01bd839a6ef5dbf08ba0d681254675f4f0be5/scripts/icesheet/update.sh#L54

Currently this does not seem equivalent to osmium_noice -  The logic is to consider anything tagged natural=* except

* the list of tags `cliff` to `coastline`
* anything that has an additional `supraglacial=yes` (that in particular applies to natural=water)

 https://github.com/joto/osmdata/blob/b5f01bd839a6ef5dbf08ba0d681254675f4f0be5/scripts/icesheet/update.sh#L58

Is this necessary?  I don't think the type attribute is actually used in the process.  I had this originally to also include the explicitly mapped glaciers in the files but that turned out to be unnecessary for usual rendering applications.

https://github.com/joto/osmdata/blob/b5f01bd839a6ef5dbf08ba0d681254675f4f0be5/scripts/icesheet/update.sh#L63

Isn't this missing a `-dsco SPATIALITE=YES`?

**Comment by joto** [comment]
The tags filtering is done in the line before:

https://github.com/joto/osmdata/blob/b5f01bd839a6ef5dbf08ba0d681254675f4f0be5/scripts/icesheet/update.sh#L50-L52

I have added `bay` to this list because there are some rather questionable taggings using `natural=bay`.

I removed the `supraglacial` stuff because there isn't a single tag of that kind on anything with a `natural` tag. If that's needed, I can put that back in.

The `type` is used later, so I think this is still needed:

https://github.com/joto/osmdata/blob/b5f01bd839a6ef5dbf08ba0d681254675f4f0be5/scripts/icesheet/update.sh#L232

> Isn't this missing a -dsco SPATIALITE=YES?

The database is not created here, so I don't think it is needed.

**Comment by imagico** [comment]
> The type is used later, so I think this is still needed:

Ah, right - but that can be changed to use the `natural` attribute (but needs to be quoted since natural is a postgres keyword).

Regarding bay and supraglacial - either would need to go with what is actually used in the database (that means neither in the Antarctic at the moment) or with what may plausibly be used that does not imply no ice (then bay is less relevant than various other tags like natural=peninsula and natural=cape).

I started a bit with a postgis version of the icesheet process - but developing this within pg_virtualenv is kind of tricky.

**Comment by joto** [comment]
> Ah, right - but that can be changed to use the natural attribute (but needs to be quoted since natural is a postgres keyword).

Yes, the quoting is annoying, especially while playing around with the data to get things figured out. Also I wanted to change your script as little as possible to make it easier to merge it again at some point.

> Regarding bay and supraglacial...

This is the place where I stumbled on the bays: https://www.openstreetmap.org/#map=13/-62.9489/-60.6409

Note the horrible mapping, for example: https://www.openstreetmap.org/way/591521220

I am open to any suggestion to filter for more or less tags here.

> developing this within pg_virtualenv is kind of tricky

I don't use pg_virtualenv when developing, I use a "normal" database connection. The scripts should all work with a normal database, too. Just set `PGDATABASE` and `PGUSER` env variables. Then you can access the database as usual.
